modules/imgcodecs/CMakeLists.txt

```diff
@@ -1,3 +1,7 @@
+if(WINRT)
+  ocv_module_disable(imgcodecs)
+endif()
```

Comment:
Extra changes are required (mostly in 3-d party codecs) to make it work. This process is underway, you can take a look at current version in [MSOpenCV/add-imgcodecs](https://github.com/MSOpenTech/opencv/tree/add-imgcodecs) branch, mind this is still WIP. Trying to make this contribution solid and at the same time respecting the ask to make it small enought imgcodecs (as well as videoio) are not included. It will likely be contributed in one of the following PRs.